modules/installation-and-upgrade/pages/install-server-unified.adoc

```diff
@@ -16,7 +16,7 @@ Before installing {susemgr}, ensure your physical or virtual machine has enough
     For more information, see xref:specialized-guides:public-cloud-guide/overview.adoc[].
 * In case of installing {susemgr} in a public cloud where a {susemgr} image is not available, it is possible to start from a {sles}{nbsp}{sles-version}{nbsp}{sp-base} and switch the base product to {susemgr}{nbsp}{productnumber}.
 
```

Comment:
I think the "for info sentence should be part of this item (to make it consistent with the previous item).  Maybe, we need + connectors.
```suggestion
* In case of installing {susemgr} in a public cloud where a {susemgr} image is not available, it is possible to start from a {sles}{nbsp}{sles-version}{nbsp}{sp-base} and switch the base product to {susemgr}{nbsp}{productnumber}.
```

Reply:
Done, well spotted, thanks.